allennlp/data/dataset_readers/sequence_tagging.py

```diff
@@ -23,18 +23,27 @@ class SequenceTaggingDatasetReader(DatasetReader):
 
     WORD###TAG [TAB] WORD###TAG [TAB] ..... \n
 
-    and converts it into a ``Dataset`` suitable for sequence tagging.
+    and converts it into a ``Dataset`` suitable for sequence tagging. You can also specify
+    alternative delimiters in the constructor.
 
     Parameters
     ----------
+    word_tag_delimiter: ``str``, optional (default=``"###"``)
+        The text that separates each WORD from its TAG.
+    word_word_delimiter: ``str``, optional (default=``"\t"``)
```

Comment:
Can we have the default delimiter just be any whitespace?  Like, have this default to `None`, which will just call `line.split()`, instead of passing in a delimiter?  I'd also probably just call this `token_delimiter` - seems simpler.